source/uri-options/uri-options.rst

```diff
@@ -269,6 +277,13 @@ pertaining to URI options apply here.
      - no
      - Amount of time spent attempting to send or receive on a socket before timing out; note that this only applies to application operations, not SDAM
 
+   * - srvServiceName
+     - any string
```

Comment:
This should restrict to strings that are valid SRV service names.  See [RFC 6335](https://datatracker.ietf.org/doc/html/rfc6335#section-5.1) and possibly [RFC 8553](https://datatracker.ietf.org/doc/html/rfc8553), though I might suggest being more lenient with length restrictions as long as the whole label with underscore is less than the 63 character limit for DNS.